### DISCUSSION TITLE
Does not seem to work with WPA2 Enterprise

### DISCUSSION THREAD

**Discussion by michelef** [chris1111/Wireless-USB-Big-Sur-Adapter#288] [General]
Hi,

I installed the driver/app and it works well with most WiFi, BUT it does not work with my company WPA2 Enterpise WiFi.
Am I doing something wrong or is it a known bug.
If the latter, any chance that can be fixed?
Thanks
Michele

**Comment by chris1111** [comment]
it does not work with WPA2, its a known issue since macOS 11
